source/shared/cpp/ObjectModel/HostConfig.cpp

```diff
@@ -538,6 +538,7 @@ CompoundButtonConfig CompoundButtonConfig::Deserialize(const Json::Value &json,
                                                                                      AdaptiveCardSchemaKey::Badge,
                                                                                      defaultValue.badgeConfig,
                                                                                      BadgeConfig::Deserialize);
+    result.borderColour = ParseUtil::GetString(json, AdaptiveCardSchemaKey::BorderColor, result.borderColour);
```

Comment:
Not able to find related Sample.json changes. Am I missing anything?

Reply:
for light mode, we do not need to add. we are using default color defined in hostConfig

Reply:
<img width="527" alt="Screenshot 2024-08-14 at 10 20 46 AM" src="https://github.com/user-attachments/assets/51a19584-1ad2-4568-b982-0d59853c75f2">

Reply:
Let's keep changes in Sample.json also. It can have default value but it explains the purpose and format of JSON.